code/modules/mob/living/simple_animal/hostile/megafauna/behemoth.dm

```diff
@@ -6,11 +6,11 @@
 	icon_living = "behemoth_axe"
 	icon_dead = "behemoth_dead"
 
-	health = 2000 //used to be 3000
-	maxHealth = 2000 //used to be 3000
+	health = 2300 //used to be 3000
+	maxHealth = 2300 //I actually see why this needs to be higher. I almost killed it with a Bozar in under 20 seconds via just holding LMB at PB range
```

Comment:
Probably put these back to 3000. This a boss guarding power armor. So far as I can tell you're doing your testing Solo against this boss and no one should be Solo fighting this boss.